SConstruct

```diff
@@ -89,16 +89,29 @@ class Data(Struct):
         """Returns the dataId for this data"""
         return dict(visit=self.visit, ccd=self.ccd)
 
-    def id(self, prefix="--id"):
+    def id(self, prefix="--id", tract=None):
         """Returns a suitable --id command-line string"""
-        return "%s visit=%d ccd=%d" % (prefix, self.visit, self.ccd)
+        if tract is None:
+            return "%s visit=%d ccd=%d" % (prefix, self.visit, self.ccd)
+        else:
+            return "%s visit=%d ccd=%d tract=%d" % (prefix, self.visit, self.ccd, tract)
```

Comment:
Very minor: I would prefer to do this as appending a `tract=%d` rather than have two lines that do almost the same thing.